lib/bing-ads-reporting/downloader.rb

```diff
@@ -14,14 +14,12 @@ def self.fetch_report(report_url)
 
     def fetch_report
       logger.debug "Downloading Bing report from: #{report_url}"
-      curl.perform
-      curl.body_str
-    end
-
-    private
-
-    def curl
-      @curl ||= Curl::Easy.new(report_url)
+      begin
+        response = HTTParty.get(report_url)
+        response.body
+      rescue StandardError => error
```

Comment:
rescue StandardError? That could rescue lots of things, and you just resolve to `nil`. If there is a check on `fetch_report` somewhere else then it *should* be ok, but maybe let's add some log with this error here.